test/FastTests/Server/Documents/Indexing/Auto/BasicAutoMapIndexing.cs

```diff
@@ -423,6 +423,12 @@ public void WriteErrors()
                     index.Start();
                     Assert.Equal(IndexRunningStatus.Running, index.Status);
 
+                    var mre = new ManualResetEvent(false);
+
+                    database.IndexStore.IndexBatchCompleted = x => { mre.Set(); };
```

Comment:
Shouldn't we attach to this before doing `index.Start()` ?

Reply:
Yes, we should. Thx

Reply:
Fixed. Force pushed